setup.cfg

```diff
@@ -2,4 +2,16 @@
 universal=1
 
 [flake8]
+exclude=*migrations*
```

Comment:
Yes, the main issue you might see is long line lengths. We manually format them or use autopep8.